modules/localgov_media/config/optional/editor.editor.wysiwyg.yml

```diff
@@ -85,39 +114,10 @@ settings:
         -
           label: 'Green ticks'
           element: '<ul class="list-checked">'
-    ckeditor5_sourceEditing:
-      allowed_tags:
-        - '<cite>'
-        - '<dl>'
-        - '<dt>'
-        - '<dd>'
-        - '<mark>'
-        - '<blockquote cite>'
-        - '<ol type>'
-        - '<h2 id>'
-        - '<h3 id>'
-        - '<h4 id>'
-        - '<h5 id>'
-        - '<h6 id>'
-        - '<a hreflang title>'
-        - '<ul type>'
-        - '<img data-entity-type data-entity-uuid data-caption>'
-        - '<drupal-media data-caption title>'
-    ckeditor5_list:
-      reversed: false
-      startIndex: true
-    ckeditor5_imageResize:
-      allow_resize: true
-    media_media:
-      allow_view_mode_override: true
     linkit_extension:
       linkit_enabled: true
       linkit_profile: default
+    media_media:
+      allow_view_mode_override: true
 image_upload:
   status: false
-  scheme: public
-  directory: inline-images
```

Comment:
this isn't in the new config. not sure if its needed?

Reply:
I didn't do anything to the image ckeditor widget. But I reckon those lines are in there as a hangover from when we used to allow images to be uploaded via wysiwyg, so it would need a `scheme: public` and `directory: inline-images`. For ages now, we have only been allowing external images via the wysiwyg which is why the line above is set to `false`. I have a feeling anytime anyone updates the  "Text Formats and Filters" in their sites and does a `config export` they'll get what we have here.